### DISCUSSION TITLE
RaspBerry Pi Zero GPIO is Relay Board Plugin needed?

### DISCUSSION THREAD

**Discussion by spinnerdog** [Dan-in-CA/SIP#284] [Q&A]
I've got a Raspberry Pi 0 2 W and a SainsMart 4 Relay board.  I can't quite figure out if the relay_board plugin is necessary.  At the moment I have the plugin turned off and some of the Relays are turned on and off as I test with the Run Once.  Is here some documentation I'm overlooking?  I'm also not clear on which pins should be connected to relays.

Thanks for helping me sort this out.

**Comment by ClemRz** [comment]
After installing the `relay_board` plugin I had my 8-relays board working as expected. Station 1 (or Master) must be wired to GPIO-0, Station 2 on GPIO-1, and so on. Make sure that you leave "Active-Low Relay" unchecked in Options > Station Handling, and make sure you leave "Use pigpio" unchecked in Options > System. Good luck!
